src/Debugger/Impl/rtvs/R/packages.R

```diff
@@ -2,18 +2,30 @@
 # Licensed under the MIT License. See LICENSE in the project root for license information.
 
 matrix.as.lists <- function(matrix.data) {
-    res <- list()
-    dn <- dimnames(matrix.data)
+    res <- base::list()
```

Comment:
You don't need to worry about it in a package, because each package gets its own, completely separate search path, and user's global environment is not on that path. So if they do something like `c <- 1`, they will break themselves, but not us.

It's only a concern for code that goes directly into an eval, because that runs in the global environment by default, and so we've had those issues with `c` (and I started pre-emptively qualifying everything else there to prevent future bugs of the same nature).